MapleServer2/Packets/FieldObjectPacket.cs

```diff
@@ -11,25 +11,25 @@ public static class FieldObjectPacket
         public static Packet LoadPlayer(IFieldObject<Player> fieldPlayer)
         {
             Player player = fieldPlayer.Value;
-            PacketWriter pWriter = PacketWriter.Of(SendOp.PROXY_GAME_OBJ)
-                .WriteByte(0x03)
-                .WriteInt(fieldPlayer.ObjectId)
-                .WriteLong(player.AccountId)
-                .WriteLong(player.CharacterId)
-                .WriteUnicodeString(player.Name)
-                .WriteUnicodeString(player.ProfileUrl)
-                .WriteUnicodeString(player.Motto)
-                .WriteByte()
-                .Write(fieldPlayer.Coord)
-                .WriteShort(player.Level)
-                .WriteShort((short) player.JobGroupId)
-                .WriteInt(player.JobId)
-                .WriteInt()
-                .WriteInt()
-                .WriteInt()
-                .WriteUnicodeString(player.HomeName)
-                .WriteInt()
-                .WriteShort();
+            PacketWriter pWriter = PacketWriter.Of(SendOp.PROXY_GAME_OBJ);
+            pWriter.WriteByte(0x03);
```

Comment:
I'm assuming this is a mode, can we handle modes properly? (mode enum, `.WriteMode`)